controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -579,7 +562,7 @@ func (r *HostedControlPlaneReconciler) deployDynatraceHttpMonitorResources(ctx c
 	}
 	if exists {
 		log.Info(fmt.Sprintf("HTTP monitor label found. Skipping creating a monitor for %s", monitorName))
-		// return nil
+		return nil
```

Comment:
This was already fixed on master right, meaning this is just because of a rebase being needed (not a big deal)?

Reply:
Correct, I merged my other branch into this branch (as this branch was opened first).
I needed the fix so I could do some manual testing.